docs/pages/get-started/installation.mdx

```diff
@@ -52,7 +52,7 @@ After installing Node.js, you can use `npx` to create a new app.
 
 ## Recommended tools
 
-- [Yarn](https://classic.yarnpkg.com/en/docs/install) for faster and more reliable dependency management. Use this instead of `npm` and `npx`.
+- [Yarn 1 (Classic)](https://classic.yarnpkg.com/en/docs/install) for faster and more reliable dependency management. Use this instead of `npm` and `npx`.
```

Comment:
```suggestion
- [Yarn Classic (v1)](https://classic.yarnpkg.com/en/docs/install) for faster and more reliable dependency management. Use this instead of `npm` and `npx`.
```

Naming nit:
![Screenshot 2024-03-16 213213](https://github.com/expo/expo/assets/719641/104c0997-6d24-4818-88e6-4149bd93ba45)